instrumentation/opentelemetry_redix/lib/opentelemetry_redix.ex

```diff
@@ -69,8 +68,12 @@ defmodule OpentelemetryRedix do
         attributes: attributes
       })
 
-    if meta[:reason] do
-      OpenTelemetry.Span.set_status(s, OpenTelemetry.status(:error, ""))
+    case meta do
```

Comment:
This could still be an `if` statement since nothing is happening in the other case.

`if meta[:kind] == :error do`